R/pkg/R/DataFrame.R

```diff
@@ -1848,3 +1848,28 @@ setMethod("crosstab",
             sct <- callJMethod(statFunctions, "crosstab", col1, col2)
             collect(dataFrame(sct))
           })
+
+
+#' This function downloads the contents of a DataFrame into an R's data.frame.
+#' Since data.frames are held in memory, ensure that you have enough memory
+#' in your system to accommodate the contents.
+#' 
+#' @title Download data from a DataFrame into a data.frame
+#' @param x a DataFrame
+#' @return a data.frame
+#' @rdname as.data.frame
+#' @examples \dontrun{
+#' 
+#' irisDF <- createDataFrame(sqlContext, iris)
+#' df <- as.data.frame(irisDF[irisDF$Species == "setosa", ])
+#' }
+setMethod(f = "as.data.frame",
```

Comment:
sorry a couple of more style nits. Based on existing code, this should be 
```
setMethod("as.data.frame",
                  signature(x = "DataFrame"),
```
(i.e. having `x = ` in the signature line and no `f=` in the name line)
Again, just to restate this isn't for functionality but just for matching the style with all the existing code